src/apps/backend/modules/otp/internal/otp-util.ts

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
+
+import { Otp } from '../types';
+
+import { OtpDB } from './store/otp-db';
+
+export default class OtpUtil {
+  public static convertOtpDBToOtp(otpDb: OtpDB): Otp {
+    const otp = new Otp();
+    otp.id = otpDb._id.toString();
+    otp.phoneNumber = otpDb.phoneNumber;
+    otp.otpCode = otpDb.otpCode;
+    otp.status = otpDb.status;
+    return otp;
+  }
+
+  public static generateOtp(): string {
+    return _.random(1000, 9999).toString(); // 4 digit OTP
```

Comment:
Can you create a constant `OTP_LENGTH` and re-write this function? This would make it easy to change the length of OTP without getting into the weeds on how its generated